### PR TITLE
browser testing role

### PR DESCRIPTION
**Pull request by jharewinton**
installs Google Chrome, Chromedriver and the virtual frame buffer, Xvfb, to allow for headless browser testing on fully featured Chrome

### CODE REVIEW

File: roles/browser-testing/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  shell: apt-get install -f -y
+
+- name: Download Chromedriver
+  get_url: url=http://chromedriver.storage.googleapis.com/2.25/chromedriver_linux64.zip dest=/tmp/chromedriver.zip
+
+- name: Unzip Chromedriver
+  unarchive: src=/tmp/chromedriver.zip dest=/usr/local/bin/
+
+- name: Enable Chromedriver
+  shell: chmod 751 /usr/local/bin/chromedriver
```
Comment (philmcmahon) [inline roles/browser-testing/tasks/main.yml]: Similarly, you could use http://docs.ansible.com/ansible/file_module.html here.

File: roles/browser-testing/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- name: Download Chrome
+  get_url: url=https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb dest=/tmp/google-chrome-stable_current_amd64.deb
+
+- name: Install Chrome - ignores errors as expect missing packages
+  shell: dpkg -i /tmp/google-chrome-stable_current_amd64.deb
+  ignore_errors: yes
+
+- name: Install missing packages for Chrome
+  shell: apt-get install -f -y
```
Comment (jharewinton) [inline roles/browser-testing/tasks/main.yml]: the ansible apt package doesn't support this particular apt command. It 'fixes' the missing packages from the chrome install, which vary from version to version of chrome

**Comment by philmcmahon** [comment]
Other than the above comments, LGTM 👍

File: roles/browser-testing/tasks/main.yml (hunk 2 above)
Comment (philwills) [inline roles/browser-testing/tasks/main.yml]: Might it be worth using the long option here, to make it more obvious for those of us less familiar with apt-get flags?

**Pull request merged**